staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/mysql/client.go

```diff
@@ -0,0 +1,92 @@
+package mysql
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/mapper-framework/pkg/common"
+)
+
+var (
+	DB *sql.DB
+)
+
+type DataBaseConfig struct {
+	MySQLClientConfig *MySQLClientConfig `json:"mySQLClientConfig"`
+}
+
+type MySQLClientConfig struct {
+	Addr     string `json:"addr,omitempty"`
+	Database string `json:"database,omitempty"`
+	UserName string `json:"userName,omitempty"`
+}
+
+func NewDataBaseClient(config json.RawMessage) (*DataBaseConfig, error) {
+	configdata := new(MySQLClientConfig)
+	err := json.Unmarshal(config, configdata)
+	if err != nil {
+		return nil, err
+	}
+	return &DataBaseConfig{
+		MySQLClientConfig: configdata,
+	}, nil
+}
+
+func (d *DataBaseConfig) InitDbClient() error {
+	password := os.Getenv("PASSWORD")
+	usrName := d.MySQLClientConfig.UserName
+	addr := d.MySQLClientConfig.Addr
+	dataBase := d.MySQLClientConfig.Database
+	dataSourceName := fmt.Sprintf("%s:%s@tcp(%s)/%s", usrName, password, addr, dataBase)
+	var err error
+	DB, err = sql.Open("mysql", dataSourceName)
+	if err != nil {
+		return fmt.Errorf("connection to %s of mysql faild with err:%v", dataBase, err)
+	}
+
+	return nil
+}
+
+func (d *DataBaseConfig) CloseSession() {
+	err := DB.Close()
+	if err != nil {
+		klog.Errorf("close mysql failed")
+	}
+
+}
+
+func (d *DataBaseConfig) AddData(data *common.DataModel) error {
+	deviceName := data.DeviceName
+	propertyName := data.PropertyName
+	tableName := deviceName + "_" + propertyName
+	datatime := time.Unix(data.TimeStamp/1e3, 0).Format("2006-01-02 15:04:05")
+
+	createSql := fmt.Sprintf("CREATE TABLE IF NOT EXISTS `%s` (id INT AUTO_INCREMENT PRIMARY KEY, ts  DATETIME NOT NULL,field TEXT)", tableName)
```

Comment:
It seems that the purpose here is to create a table. Is it better to name it createTable?

Reply:
ok